net/speedtest/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+version             1.1.1.84
+revision            0
+
+name                speedtest
+categories          net
+platforms           darwin
```

Comment:
This line can be omitted. (It is the default.)
```suggestion
```

Reply:
Resolved in b5c45453f88a930d37fc00cb64eaf0283cf5e57c.